pkg/zfs/zfs_util.go

```diff
@@ -446,6 +485,37 @@ func SetDatasetLegacyMount(vol *apis.ZFSVolume) error {
 	return err
 }
 
+func UnshareZFSDataset(vol *apis.ZFSVolume) error {
+	// ZFS datasets cannot be destroyed without being unshared first, as without
+	// the rpcbind will still hold handles on the ds, resulting in a is busy err
+	// on destroy, unmount etc
+	volume := vol.Spec.PoolName + "/" + vol.Name
+	mntPnt := "/" + volume
+
+	var UnshareVolArg []string
+	UnshareVolArg = append(UnshareVolArg, "-u", "192.168.0.0/16:"+mntPnt)
```

Comment:
harcoded ip?

Reply:
good that you spotted that ... "quick fix" and then forgotten. I'll fix that